recipes-graphics/wayland/wayland-protocols_1.38.bb

```diff
@@ -0,0 +1,30 @@
+SUMMARY = "Collection of additional Wayland protocols"
```

Comment:
this will override the core provided recipe and we may fail yocto project compatible checks. I wonder if there is a better solution for this.